app/actions/usersActivity.js

```diff
@@ -0,0 +1,24 @@
+import { USERS_ACTIVITY } from './actionsTypes';
+
+export function addUserActivity(username, activity, rid, tmid) {
+	return {
+		type: USERS_ACTIVITY.ADD,
+		roomId: tmid || rid,
+		username,
+		activity
+	};
+}
+
+export function clearUserActivity(username, roomId) {
+	return {
+		type: USERS_ACTIVITY.CLEAR_USER_ACTIVITY,
+		roomId,
+		username
+	};
+}
+
+export function removeRoomUsersActivity() {
```

Comment:
Should we send a `rid`?
Should we rename this to `removeAllUsersActivity` or something like that?

Can you test this with multiple rooms and discussions?